src/python/pants/backend/docker/util_rules/docker_build_context_test.py

```diff
@@ -155,7 +155,7 @@ def test_pants_hash(rule_runner: RuleRunner) -> None:
                 "stage0": "latest",
             },
             "build_args": {},
-            "pants": {"hash": "fd19488a9b08a0184432762cab85f1370904d09bafd9df1a2f8a94614b2b7eb6"},
+            "pants": {"hash": "32b02dd86d1ad0169fa02a28583b70e8834ce22a5fccb5e9de422315055de6a0"},
```

Comment:
This warrants a ~big note in the changelog as it will break the stability guarantee from the note here https://www.pantsbuild.org/docs/tagging-docker-images#string-interpolation-using-placeholder-values

I also acknowledge that I failed to verify this hash thoroughly enough to catch that build args where in fact not affecting the hash value, so that was a bug.

Reply:
I don't think that breaks the stability guarantee though, right? Because it says we may break things between releases?

Reply:
Ah yes, bad phrasing. I meant that we break the stability of the hashing but indeed we don't break the guarantee as long as we don't pick this back to a stable release branch.

Reply:
But we still want to make users aware of this when upgrading.

Reply:
How do you suggest we ensure this makes it into the notes?

Reply:
Add a section to the "what's new" section in this PR maybe?
https://github.com/pantsbuild/pants/blob/57db6f214c9fce7a255c4f167d32aecad78649c7/src/python/pants/notes/2.16.x.md?plain=1#L1-L3

Reply:
AH yeah good idea.